### PR TITLE
Update the explanation to link a google analytics account since google has revamped the interface (again...)

### PR DESCRIPTION
**Pull request by carakas**
## Type

- Enhancement

## Pull request description

> Describe what your pull request will fix / add / …

Google has once again updated the developer console.
This will update the translations for the Analytics module so the guide still works

### CODE REVIEW

File: src/Backend/Modules/Analytics/Installer/Data/locale.xml [hunk 1]
```diff
@@ -188,17 +188,21 @@
...
           </p>
           <p>Toegangsgegevens aanmaken voor Fork CMS.</p>
           <p>
             <ol>
-              <li>Kies in de zijbalk aan de linkerkant <strong>Credentials.</strong></li>
-              <li>Klik op deze pagina op New credentials onder de credentials tab.</li>
-              <li>Kies <strong>Service account</strong> in de dropdown.</li>
+              <li>Kies in de zijbalk aan de linkerkant <strong>Inloggegevens</strong>.</li>
+              <li>Klik op <strong>Inloggegevens maken</strong> en kies <strong>Serviceaccountsleutel</strong> in de dropdown.</li>
+              <li>Maak een nieuw serviceaccount aan met de rol <strong>Project - Editor</strong> en <strong>P12</strong> as type sleutel.</li>
```
Comment (StijnVrolijk) [inline src/Backend/Modules/Analytics/Installer/Data/locale.xml]: typo: as -> als

File: src/Backend/Modules/Analytics/Installer/Data/locale.xml [hunk 2]
```diff
@@ -188,17 +188,21 @@
             <ol>
               <li>Ga naar de <a href="https://console.developers.google.com/" target="_blank">Google Developers Console</a>.</li>
               <li>Zorg dat je aangemeld bent met een Google account die toegang heeft op de gewenste analytics account.</li>
-              <li>Kies een project of maak een nieuw aan.</li>
-              <li>Klik in de zijbalk op <strong>APIs &amp; auth</strong>. Kies daaronder <strong>APIs</strong>. Kies <strong>Enabled APIs</strong> om te zien welke API's er ingeschakeld zijn. Controleer of the Analytics API tussen de ingeschakelde API's staat. Indien de Analytics API niet aanwezig is, klik dan op de <strong>Enable API</strong> knop voor deze API.</li>
+              <li>Kies een project in de header of maak een nieuw aan.</li>
+              <li>Klik in de zijbalk op <strong>Bibliotheek</strong></li>
```
Comment (StijnVrolijk) [inline src/Backend/Modules/Analytics/Installer/Data/locale.xml]: I'd make this: 'Klik op Bibliotheek in de zijbalk'

File: src/Backend/Modules/Analytics/Installer/Data/locale.xml [hunk 3]
```diff
@@ -188,17 +188,21 @@
...
+              <li>Ga naar de <strong>Analytics API</strong> door er op te klikken in de <strong>Andere populaire API's</strong> categorie of door het te typen in de zoekbalk.</li>
+              <li>Als je de API not niet geactiveerd hebt kan je dit doen door onder de header op <strong>INSCHAKELEN</strong> te klikken.<li>
```
Comment (StijnVrolijk) [inline src/Backend/Modules/Analytics/Installer/Data/locale.xml]: not -> nog

File: src/Backend/Modules/Analytics/Installer/Data/locale.xml [hunk 4]
```diff
@@ -188,17 +188,21 @@
...
             <ol>
-              <li>Kies in de zijbalk aan de linkerkant <strong>Credentials.</strong></li>
-              <li>Klik op deze pagina op New credentials onder de credentials tab.</li>
-              <li>Kies <strong>Service account</strong> in de dropdown.</li>
+              <li>Kies in de zijbalk aan de linkerkant <strong>Inloggegevens</strong>.</li>
+              <li>Klik op <strong>Inloggegevens maken</strong> en kies <strong>Serviceaccountsleutel</strong> in de dropdown.</li>
+              <li>Maak een nieuw serviceaccount aan met de rol <strong>Project - Editor</strong> en <strong>P12</strong> as type sleutel.</li>
               <li>Download na het opslaan het gegenereerde certificaat (p12 file).</li>
+              <li>Ga terug naar de <strong>Inloggegevens</strong> pagina en klik op <strong>Serviceaccounts beheren</strong>.</li>
+              <li>Kopieer de <strong>Serviceaccount-ID</strong> van het net aangemaakte account. Het zou er ongeveer als volgt moeten uitzien <code>name@spheric-passkey-123456.iam.gserviceaccount.com</code></li>
```
Comment (StijnVrolijk) [inline src/Backend/Modules/Analytics/Installer/Data/locale.xml]: 'de account', not 'het account'

File: src/Backend/Modules/Analytics/Installer/Data/locale.xml [hunk 5]
```diff
@@ -211,17 +215,21 @@
             <ol>
               <li>Go to the <a href="https://console.developers.google.com/" target="_blank">Google Developers Console</a>.</li>
               <li>Make sure you're logged in with a Google account that has access to the wanted Analytics account.</li>
-              <li>Select a project, or create a new one.</li>
-              <li>In the sidebar on the left, expand <strong>APIs &amp; auth</strong>. Next, click <strong>APIs</strong>. Select the <strong>Enabled APIs</strong> link in the API section to see a list of all your enabled APIs. Make sure that the Google Analytics API is on the list of enabled APIs. If you have not enabled it, select the API from the list of APIs, then select the <strong>Enable API</strong> button for the API.</li>
+              <li>Select a project in the header, or create a new one.</li>
+              <li>Click on <strong>Library</strong> in the sidebar on the left.</li>
+              <li>Go to the <strong>Analytics API</strong> page by clicking on it in the <strong>Other popular API's</strong> category or typing it in the seach bar.</li>
```
Comment (StijnVrolijk) [inline src/Backend/Modules/Analytics/Installer/Data/locale.xml]: seach -> search

Comment (carakas) [inline src/Backend/Modules/Analytics/Installer/Data/locale.xml]: fixed in acd9bbd

File: src/Backend/Modules/Analytics/Installer/Data/locale.xml (hunk 1 above)
Comment (carakas) [inline src/Backend/Modules/Analytics/Installer/Data/locale.xml]: fixed in acd9bbd

File: src/Backend/Modules/Analytics/Installer/Data/locale.xml (hunk 3 above)
Comment (carakas) [inline src/Backend/Modules/Analytics/Installer/Data/locale.xml]: fixed in acd9bbd

File: src/Backend/Modules/Analytics/Installer/Data/locale.xml (hunk 4 above)
Comment (carakas) [inline src/Backend/Modules/Analytics/Installer/Data/locale.xml]: I don't get what is wrong with this one

**Comment by StijnVrolijk** [comment]
@carakas there's a typo in your last commit message. From should be for ;)

**Pull request merged**